tds.go

```diff
@@ -1052,6 +1053,9 @@ func parseConnectParams(dsn string) (connectParams, error) {
 	p.hostInCertificate, ok = params["hostnameincertificate"]
 	if !ok {
```

Comment:
Else here becomes a double-negative, can you change it to:
```suggestion
	if ok {
		p.hostInCertificateProvided = true
	} else {
		p.hostInCertificate = p.host
		p.hostInCertificateProvided = false	
	}
```
Everything else looks good to me.